tests/debugger/do_backtrace.expected

```diff
@@ -14,10 +14,15 @@ out: function test
 Stopped at tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 (jerry-debugger) s
 Stopped at tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
+(jerry-debugger) bt 1 2
+Frame 0: tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
```

Comment:
Is this intentional? I thought this should be `Frame 1: tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)`